backend/routers/index.js

```diff
@@ -6,12 +6,38 @@ const PORT = 4000;
 app.use(express.json());
 
 app.get('/', (req, res) => {
-  res.status(200).send('Hellow world');
+  res.status(200).send('Hello world');
 });
 
 app.use((err, req, res, next) => {
   res.status(500).send('Something broke!');
   next(err);
 });
 
+const dummyData = {
```

Comment:
please don't call it like this, it's obvious that it's just mocked data. rename it to myTroops, or something similar